migrations/20230307121010-add-locations-table.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, DataTypes) {
+    await queryInterface.createTable('Locations', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER,
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      address: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      country: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      structured: {
+        type: DataTypes.JSONB,
+        allowNull: true,
+      },
+      geoLocationLatLong: {
+        type: DataTypes.JSONB,
+        allowNull: true,
+      },
```

Comment:
I haven't benchmarked, but intuitively I would say that querying this as JSONB will be much slower than querying two (lat: FLOAT, long: FLOAT) columns or a Postgres `Point` (as suggested in https://github.com/opencollective/opencollective/issues/3494)

Reply:
Makes sense. Got stuck on some Typescript issues I were not able to resolve quickly (extending DataTypes to include `POINT`) but might consider it a future enhancement, to be able to focus on some other projects this cycle, added this in a separate PR here: https://github.com/opencollective/opencollective-api/pull/8704.